### ISSUE TITLE
Afwall+ Compatibility

### ISSUE THREAD

**Issue by HuiiBuh** [pi-hole/pi-hole#3025]
**In raising this issue, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

I am not at all familiar with the source code of pi-hole, but understand the technical principles that stand behind the project.

---

[Afwall+](https://github.com/ukanth/afwall/) Is a app for the smartphone that allows users to block the internet access of certain apps using IP tables. The expected behaviour is that pi-hole and Afwall+ work separated, because pi-hole only provides the DNS and Afwall+ edits the iptables. 
So in my opinion they should not interfere with each other.

The problem is that every time that I connect to my network (which uses pi-hole as DNS) **.org** domains cannot be accessed. I tried it on two different smartphones (one running a custom ROM on android 9 and the other one running a different custom ROM on android 10). As soon as I disables pi-hole or Afwall+ **.org** domains can be accessed again. So far I only notices the blocking of **.org** domains. So there seems to be some kind of connection between pi-hole and Afwall+. 
Networks without pi-hole installed and Afwall+ enabled are no problem and vice versa.

So I cannot access `wikipedia.org` but `wikipedia.de` is still accessible.
This problem was also reported by another use on the Afwall+ repo ([here](https://github.com/ukanth/afwall/issues/999))

**Steps to reproduce:**

+ Install pi-hole
+ Install Afwall+ and activate it to block the internet access of certain apps

### And thank you for maintaining such a great project!

**Comment by DL6ER** [comment]
Use a separate device (laptop, computer, ...) to browse to http://pi.hole/admin/taillog.php

You should see a page which lists all DNS traffic in real-time. Now, connect your phone (first without Afwall+) and query some domain. Does the query appear on this page? If yes, good. If no, then your phone is likely not using your Pi-hole at all (unlikely from your description).

Next, enable your Afwall+ app and query said domain again. Does it still appear on the live Pi-hole web interface page?

**Comment by HuiiBuh** [comment]
Hi thanks for the advice. 
I did what you said and both times `wikipedia.org` gets queried. So the phone is using pihole.
And if I use nslookup on my phone with Afwall+ enabled to resolve the IP adress the address gets resolved. But if I trie to reach any *.org* domain with the browser I get a **NAME_NOT_RESOLVED** error. 
This is really a strange bug and I still have no Idea where the problem is. After all termux, which I used to run the nslookup command, and the browser (brave and firefox) are restricted by the same rules.
